.github/workflows/release.yml

```
@@ -183,10 +183,63 @@ jobs:
         with:
           wait-timeout-minutes: 5
 
+  build_linux:
+    name: Build Release DEB
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```

actionlint (1.7.7)

the runner of "actions/checkout@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)